Cargo.toml

```diff
@@ -156,7 +156,6 @@ syn = "2.0"
 sysinfo = "0.32"
 tabled = { version = "0.16.0", default-features = false }
 tempfile = "3.14"
-terminal_size = "0.4"
```

Comment:
Since `terminal_size` is still used for two crates, we should probably keep this here to synchronize the versions.

Reply:
ok